src/components/loader/loader.mdx

```diff
@@ -68,6 +68,13 @@ please remember to pass the `isInsideButton` prop to the `Loader` component.
 
 <Canvas of={LoaderStories.InsideButtons} />
 
+### Accessible Example 
+
+This example demonstrates how to ensure that the `Loader` component is accessible when used inside of the `Button` component. 
```

Comment:
**suggestion**: it might be a good idea to also advise caution about conditionally rendering, given live regions need to exist on the page before content changes are made.

Reply:
Yep that is a very good point. Completely forgot to add that in.